graylog2-server/src/test/java/org/graylog/plugins/pipelineprocessor/functions/FunctionsSnippetsTest.java

```diff
@@ -1134,7 +1134,82 @@ void fieldPrefixSuffix() {
 
     @Test
     void keyValue() {
-        final Rule rule = parser.parseRule(ruleForTest(), true);
+        final var r = """
```

Comment:
I would prefer if we not inlined some of the rules while leaving others in the files.
One motivation to have them in files was not having to escape quotes, which is somewhat obsolete now that Java has text blocks, but I'd still retain the consistency of having all rule source in a separate file, especially if they are long.

Reply:
reverted the change